dll/win32/shell32/folders/CFSFolder.h

```diff
@@ -117,6 +117,11 @@ class CFSFolder :
 
     protected:
         HRESULT WINAPI GetCustomViewInfo(ULONG unknown, SFVM_CUSTOMVIEWINFO_DATA *data);
+
+    public:
+        // Helper functions shared with CDesktopFolder
+        static HRESULT GetFSColumnDetails(UINT iColumn, SHELLDETAILS &sd);
+        static HRESULT GetDefaultFSColumnState(UINT iColumn, SHCOLSTATEF &csFlags);
```

Comment:
Do we want to move those 2 function declarations upwards into the existing public sections?
I do see that they are static, and the others are not. Still the question.

Reply:
I placed them alone not just because they are static. They are public helper functions other folder implementations inside shell32 are allowed to use. Specifically for CDesktopFolder but RegFolder might also want to know at some point in the future.